Core/Inc/cockpit/feedback.h

```diff
@@ -0,0 +1,74 @@
+#ifndef FEEDBACK_H
+#define FEEDBACK_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+/*!
+ * \brief Possible return codes for feedback functions
+ */
+enum FeedbackState {
+    FEEDBACK_STATUS_HIGH, //!< Feedback line is high
+    FEEDBACK_STATUS_LOW,  //!< Feedback line is low
+    FEEDBACK_STATUS_ERROR //!< Error in reading feedback button states
+};
+
+/*!
+ * \brief Enum for identifying feedback lines
+ */
+enum FeedbackName {
+    FEEDBACK_NAME_STEERING_WHEEL,
+    FEEDBACK_NAME_MUSHROOM_BEFORE,
+    FEEDBACK_NAME_MUSHROOM_AFTER,
+    FEEDBACK_NAME_COUNT
+};
+
+/*!
+ * \brief Possible return codes for feedback initialization function
+ */
+enum FeedbackRC {
+    FEEDBACK_RC_ERROR,
+    FEEDBACK_RC_OK
+};
```

Comment:
As for the convention the enum member prefix should be the enum name but since the return code is often used in most of the projects it can be written in a shorter form (i.e. `FEEDBACK_RC` in this case):
```suggestion
enum FeedbackReturnCode {
    FEEDBACK_RC_OK,
    FEEDBACK_RC_ERROR
};
```